### ISSUE TITLE
keras_contrib

### ISSUE THREAD

**Issue by Alukassh** [simontomaskarlsson/CycleGAN-Keras#16]

**Comment by Alukassh** [comment]
I just follow the command "pip install git+https://www.github.com/farizrahman4u/keras-contrib.git" to install keras_contrib successfully, but can't import InstanceNormalization yet. Does anyone meet similar problem or have any suggesstion? Thanks in advance!

**Comment by mha-py** [comment]
I solved it with by replacing
`from keras_contrib.layers.normalization import InstanceNormalization, InputSpec`
by
```
from keras_contrib.layers.normalization.instancenormalization import InstanceNormalization
from keras.layers import InputSpec
```
but im not sure yet if inputspec is imported the right way

**Comment by kpagels** [comment]
> I just follow the command "pip install git+https://www.github.com/farizrahman4u/keras-contrib.git" to install keras_contrib successfully, but can't import InstanceNormalization yet. Does anyone meet similar problem or have any suggesstion? Thanks in advance!

I did the same as @mha-py by replacing it with the following:

    from tensorflow.python.keras.layers import InputSpec
    from keras_contrib.layers.normalization.instancenormalization import InstanceNormalization

But it seems like you solved it @Alukassh ?
